src/gtest/test_sidechain_certificate_quality.cpp

```diff
@@ -286,7 +286,7 @@ TEST_F(SidechainsMultipleCertsTestSuite, Cert_LowerQuality_DifferentEpoch_UndoDa
     EXPECT_TRUE(initialScState == revertedSidechain);
 }
 
-TEST_F(SidechainsMultipleCertsTestSuite, V2Cert_HigherQuality_SameEpoch_SidechainIsNotUpdated) {
+TEST_F(SidechainsMultipleCertsTestSuite, V2Cert_SameEpoch_SidechainIsNotUpdated) {
```

Comment:
I would rename this test from `V2Cert_SameEpoch_SidechainIsNotUpdated` to `V2Cert_SameEpoch_SameQuality_SidechainIsNotUpdated` for better understanding and coherence with other tests names

Reply:
Updated in dfeb52fab30e3b941b44631be6cdf8a95f317a67